docs/build_linux.md

```diff
@@ -2,46 +2,74 @@
+This tutorial was made with Ubuntu Xenial Xerus (16.04) and Arch Linux in mind, however you should be able to adapt this guide to other versions or distributions if necessary.
 
 ## Installing Dependencies
 
-There are a few things that must be installed in order to build Pencil2D. In this section we will go over the installation of each of these components in detail. We describe both graphical and command-line methods of installation, choose whichever one you feel more comfortable with (and if you don't know the difference, choose graphical).
+There are a few things that must be installed in order to build Pencil2D. In this section we will go over the installation of each of these components in detail. For Ubuntu we describe both graphical and command-line methods of installation, choose whichever one you feel more comfortable with (and if you don't know the difference, choose graphical). For Arch Linux we describe only the command-line method since that is what most Arch users are used to.
 
 ### Qt
 
 Pencil2D relies on the Qt application framework so you must install it before you can successfully build the program.
 
-#### Graphical method
+#### Ubuntu
+
+##### Graphical method
 
 - Go to the [Qt Downloads](https://www.qt.io/download-open-source/) and download the *Qt Online Installer for Linux*.
 - Executing this file will start the Qt installer application. If you can't open it right away, you may have to right click on it and go to *Properties*, then in the *Permissions* tab select *Allow executing file as program* and then try opening it again.
 - Click Next. It will give you the option to log in with your Qt developer account. You can skip this if you don't have an account and do not want to register.
 - Next specify a location for Qt, put it somewhere you can find it in case you ever need to navigate to the Qt files manually.
-- Next you can select the components you wish to install. At the very least you should have Desktop gcc selected under the latest Qt version. Also make sure Qt Creator under the Tools section is being installed (at the time of writing there is no option to uncheck this, but it's worth double checking!)
+- Next you can select the components you wish to install. At the very least you should have Desktop GCC selected under the latest Qt version. Also make sure Qt Creator under the Tools section is being installed (at the time of writing there is no option to uncheck this, but it's worth double checking!)
 - Agree to the license and begin the installation. It will take a long time to download all of the files, so be patient. When the installation is complete, press Done and it will launch Qt Creator for you.
 
-#### Command-line method
+##### Command-line method
 
 Pencil2D must be built with the Qt 5 framework. To install Qt 5, run this command:
 
-    sudo apt install qt5-default qt5-qmake qtdeclarative5-dev qttools5-dev libqt5xmlpatterns5-dev libqt5svg5-dev qtmultimedia5-dev
```

Comment:
I think qttools5-dev is needed for handling the translation files. qtdeclarative5-dev is/was probably needed for something too, but I don't remember what.

Reply:
Hmm are you sure? qttools5-dev only contains some includes and libraries. Maybe qttools5-dev-tools instead, which contains linguist and such? Also qtdeclarative5-dev only contains stuff for Qml and Qt Quick which I’m pretty sure we don’t use. See https://packages.ubuntu.com/xenial/amd64/qtdeclarative5-dev/filelist, https://packages.ubuntu.com/xenial/amd64/qttools5-dev/filelist, https://packages.ubuntu.com/xenial/amd64/qttools5-dev-tools/filelist

Reply:
Oh sorry, I was looking at the docs for qttools5-dev-tools. You're right, I'll double check, but neither of these look necessary.

Reply:
Just tested this. I can confirm that the build worked fine without either of these. 👍